Components/ImageOfPokemon.tsx

```diff
@@ -1,12 +1,22 @@
 import React from "react";
 import { Image } from "react-native";
 
-export const ImageOfPokemon = ({ name }: { name: string }) => (
+export const ImageOfPokemon = ({
+  name,
+  isFullImage,
+}: {
+  name: string;
+  isFullImage: boolean;
+}) => (
```

Comment:
Prznieś sobie to do osobnego interfejsu:
```suggestion
}: ImageOfPokemonProps) => (
```